src/main/java/frc/robot/Robot.java

```diff
@@ -12,28 +12,34 @@
 
 package frc.robot;
 
-import frc.bionic.UserInterfaceElement;
-import frc.bionic.swerve.AbstractDrivetrain;
-import edu.wpi.first.wpilibj.TimedRobot;
+import org.opencv.core.RotatedRect;
+
 import edu.wpi.first.networktables.NetworkTableEntry;
-import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
-import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.geometry.Translation2d;
 import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
 import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import frc.bionic.Conversion;
+import frc.bionic.TrajectoryFollow;
+import frc.bionic.TrajectoryUtils;
+import frc.bionic.UserInterfaceElement;
+import frc.bionic.swerve.AbstractDrivetrain;
 
 public class Robot extends TimedRobot {
-  private AbstractDrivetrain    drivetrain;
+  private AbstractDrivetrain drivetrain;
 
   // Shuffleboard-related
-  NetworkTableEntry         sb_robot_type;
-
+  NetworkTableEntry sb_robot_type;
 
   @Override
   public void robotInit() {
-    ShuffleboardTab           tab;
+    ShuffleboardTab tab;
 
     tab = Shuffleboard.getTab("Robot Setup");
-    sb_robot_type = tab.addPersistent("Drivetrain Type",  "Enter 'peyton' or 'team4909'").getEntry();
+    sb_robot_type = tab.addPersistent("Drivetrain Type", "Enter 'peyton' or 'team4909'").getEntry();
 
     SmartDashboard.putBoolean("Override", false);
     SmartDashboard.putBoolean("ZERO", false);
```

Comment:
Temporary dashboard stuff? If so, let's delete it. If not, can we convert it to shuffleboard style so we can easily tell what's intended as permanent dashboard stuff and what's just temporary?